docs/ref/csrf.txt

```diff
@@ -462,3 +472,30 @@ A number of settings can be used to control Django's CSRF behavior:
 * :setting:`CSRF_FAILURE_VIEW`
 * :setting:`CSRF_HEADER_NAME`
 * :setting:`CSRF_TRUSTED_ORIGINS`
+
+Frequently Asked Questions
+==========================
+
+Is posting an arbitrary CSRF token pair (cookie and POST data) using Firebug, Chrome dev tools, Burp Suite, etc. a vulnerability?
+---------------------------------------------------------------------------------------------------------------------------------
+
+No, this is by design. The comparison between the cookie token and the POST
```

Comment:
The second sentence strikes me as providing little to no information; I would focus more on something along the lines of: "Without a man-in-the-middle attack, there is no way for an attacker to send the csrftoken cookie to the victims browser, so a successful attack would need to obtain it via XSS or similar in which case you usually do not need CSRF attacks anymore"